cmd/relay/main.go

```diff
@@ -3,19 +3,29 @@ package main
 import (
 	"os"
 
+	"github.com/puppetlabs/relay/pkg/config"
+	"github.com/puppetlabs/relay/pkg/format/error"
 	"github.com/spf13/cobra"
 )
 
 func main() {
 	cmd := &cobra.Command{
-		Use:   "relay",
-		Short: "Relay by Puppet.",
-		Args:  cobra.MinimumNArgs(1),
+		Use:           "relay",
+		Short:         "Relay by Puppet.",
```

Comment:
Total nit, but it would be good to standardize on these being sentences (ending with ".") or not. I'd be inclined to say that we shouldn't use full sentences in the short description.